notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -5776,19 +5776,19 @@ orphanhood [Young Elizabeth was adopted into Baron Alphonse Frankenstein's home
 grief [Baron Alphonse Frankenstein was utterly devastated when his wife died in childbirth, and he was again grief-stricken when his beloved grandson Willie was found murdered. Victor didn't take it well when the monster ripped out his wife's heart right before his very eyes.]
 father and son [Baron Alphonse Frankenstein and his son Victor Frankenstein. Frankenstein's monster referred to his maker Victor Frankenstein as his father after Victor had died.]
 brother and sister [Victor and his step-sister Elizabeth fell in love and eventually married. Their relationship as brother and sister, however, was not much explored. For the purpose of the plot they were effectively a couple who were engaged to be married.]
-what it is like in university [Victor attended university lectures in Ingolstadt.]
+what it is like at university [Victor attended university lectures in Ingolstadt.]
 the thirst for knowledge [Victor was described as having been "hungry for knowledge" since he was a boy.]
 facing an epidemic [Cholera was spreading like wildfire in Ingolstadt.]
 love vs. the public good [When faced with the choice, Victor elected to continue with his research into conquering death over his fiancée Elizabeth. However, the pair later reunited and got married.]
 remorse [Victor came to regret his decision to reanimate a corpse (actually a patchwork of multiple dead people) upon seeing the grotesque and flawed outcome of his work.]
 husband and wife [Victor and Elizabeth lived briefly and happily as husband and wife before the monster murdered Elizabeth. A young peasant and his wife were struggling to survive the winter until Frankenstein's monster helped them to secure their winter food supply.]
 coping with a disability [The Grandfather was blind.]
 educating a child [A peasant mother was teaching her young daughter how to read.]
-Christmas traditions [The peasant family left a Christmas present addressed to "the Good Spirit of the Forest".]
-coping with the death of a child [The murder of his beloved grandson Willie left Baron Alphonse Frankenstein bedridden with grief.]
+Christmas tradition [The peasant family left a Christmas present addressed to "the Good Spirit of the Forest".]
+death of a child [The murder of his beloved grandson Willie left zBaron Alphonse Frankenstein bedridden with grief.]
```

Comment:
Typo: zBaron, I assume..

Reply:
Hum... I think you must be looking at the dev branch version of this story. It's fixed on the master branch. Need to figure out how to get the dev branch up to date.